ParisBNF/abb/BNFabb119.xml

```diff
@@ -0,0 +1,547 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BNFabb119" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>New Testament Epistles, Acts of the Apostles, Revelation of John</title>
+            <editor role="cataloguer" key="DN"/>
+            <editor role="generalEditor" key="AB"/>
+            <funder>Akademie der Wissenschaften in Hamburg</funder>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <pubPlace>Hamburg</pubPlace>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                  licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+              <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+
+                  <idno facs="https://archivesetmanuscrits.bnf.fr/ark:/12148/cc109121r">BnF Éthiopien d'Abbadie 119</idno>
+
+                    </msIdentifier>
+
+                    <msContents>
+                        <summary/>
+
+                  <msItem xml:id="ms_i1">
+                            <locus from="1r" to="114ra"/>
+                            <title type="incomplete" ref="LIT3505PaulineEpis">Pauline Epistles</title>
+                            <textLang mainLang="gez"/>
+                            <note>Most of the traditional
+                              introductions are placed after <ref target="#ms_i2"/>.</note>
+
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="1r"/>
+                                <title type="complete" ref="LIT3515Epistle">Epistle to the Romans</title>
+                                <textLang mainLang="gez"/>
+
+                                <msItem xml:id="ms_i1.1.1">
+                                    <locus from="1r"/>
+                                    <title type="complete" ref="LIT3515Epistle#Introduction"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                </msItem>
```

Comment:
delete the closing /msItem to correct nesting, 1.1.1 and 1.1.2 should be nested together under 1.1

Reply:
the closing /msItem should be added instead before 1.2 (after 1.15.1)